erpnext/patches/v13_0/set_stock_valuation_method.py

```diff
@@ -4,4 +4,11 @@ def execute():
     stock_settings = frappe.get_doc('Stock Settings')
     if not (stock_settings.valuation_method):
         stock_settings.valuation_method = 'FIFO'
-    stock_settings.save()
\ No newline at end of file
+    stock_settings.save()
+
+    #Updating valuation method for exisitng items
+    frappe.reload_doc("stock", "doctype", "item")
+    items = frappe.get_all('Item', fields = ['name', 'valuation_method'])
+    for item in items:
+        if not (item.valuation_method):
+            frappe.db.set_value("Item", item.name, "valuation_method", "FIFO", update_modified=False)
```

Comment:
- Use SQL to set it as FIFO where value doesn't exist.
- We can save time taken in fetching all the items and then iterating over them